packages/neuron-wallet/src/models/asset-account-info.ts

```diff
@@ -183,6 +184,14 @@ export default class AssetAccountInfo {
         hashType: process.env.TESTNET_NFT_SCRIPT_HASH_TYPE! as ScriptHashType
       }
     }
+    this.defaultLockScriptInfo = {
```

Comment:
It can be imported from https://github.com/nervosnetwork/neuron/blob/1e38bf860b46e5358081aefb2b816ceb52d32cb9/packages/neuron-wallet/src/models/system-script-info.ts#L35-L43

Reply:
This is an async method, which is inconvenient to use in the constructor.